### PR TITLE
Fix background images for drafts and confirmation pages

### PR DESCRIPTION
**Pull request by davodesign84**

### CODE REVIEW

File: packages/kauri-web/components/containers/EditProfile/View.js [hunk 1]
```diff
@@ -1,8 +1,17 @@
+<<<<<<< HEAD
```
Comment (rej156) [inline packages/kauri-web/components/containers/EditProfile/View.js]: Lol

Comment (davodesign84) [reply packages/kauri-web/components/containers/EditProfile/View.js]: ahahah yeah XD

**Comment by rej156** [comment]
Lgtm

File: packages/kauri-web/components/containers/EditProfile/View.js (hunk 1 above)
Comment (davodesign84) [reply packages/kauri-web/components/containers/EditProfile/View.js]: You're too quick man, go back to lunch.

**Comment by davodesign84** [comment]
Totally forgot that InReviewArticle is used for approving articles..

**Pull request merged**